src/main/proto/wfa/measurement/internal/kingdom/exchange_step_attempt_details.proto

```diff
@@ -0,0 +1,63 @@
+// Copyright 2021 The Cross-Media Measurement Authors
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+
+syntax = "proto3";
+
+package wfa.measurement.internal.kingdom;
+
+import "google/protobuf/timestamp.proto";
+
+option java_package = "org.wfanet.measurement.internal.kingdom";
+option java_multiple_files = true;
+
+// Stores metadata about an ExchangeStepAttempt.
+message ExchangeStepAttemptDetails {
+  enum State {
```

Comment:
how is this state different than the `ExchangeStepAttempt` state? https://github.com/world-federation-of-advertisers/cross-media-measurement/pull/10/files#diff-70da6a8230c7350d91d41dd49c001d173f912fc7553fc81b3ddfcab8cb711879R47

Reply:
> > _[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/7)_ status: all files reviewed, 1 unresolved discussion (waiting on @AlekseiS, @oliver-amzn, @stevenwarejones, and @zasadnyy)
> 
> _[src/main/proto/wfa/measurement/internal/kingdom/exchange_step_attempt_details.proto, line 26 at r3](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/7#-MXdGU72doxHXBexv3XN-r3-26:-MYCLUGC15zujKPYCfR8:b-3wl283) ([raw file](https://github.com/world-federation-of-advertisers/cross-media-measurement/blob/389eb1ee09d08545c460fd4be43fe496fd7dc6f0/src/main/proto/wfa/measurement/internal/kingdom/exchange_step_attempt_details.proto#L26)):_
> 
> Previously, stevenwarejones (Steven Ware Jones) wrote…
> You're looking into the future... that CL fixes up some issues in this one (and builds on it). Somewhere in the UI it should say that that pull request is to be merged into the branch hosting this one. GitHub is pretty dreadful at dependent PRs compared to internal tooling so maybe this isn't the right way to do it?



> > _[Reviewable](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/7)_ status: all files reviewed, 1 unresolved discussion (waiting on @AlekseiS, @oliver-amzn, @stevenwarejones, and @zasadnyy)
> 
> _[src/main/proto/wfa/measurement/internal/kingdom/exchange_step_attempt_details.proto, line 26 at r3](https://reviewable.io/reviews/world-federation-of-advertisers/cross-media-measurement/7#-MXdGU72doxHXBexv3XN-r3-26:-MYCLUGC15zujKPYCfR8:b-3wl283) ([raw file](https://github.com/world-federation-of-advertisers/cross-media-measurement/blob/389eb1ee09d08545c460fd4be43fe496fd7dc6f0/src/main/proto/wfa/measurement/internal/kingdom/exchange_step_attempt_details.proto#L26)):_
> 
> Previously, stevenwarejones (Steven Ware Jones) wrote…
> You're looking into the future... that CL fixes up some issues in this one (and builds on it). Somewhere in the UI it should say that that pull request is to be merged into the branch hosting this one. GitHub is pretty dreadful at dependent PRs compared to internal tooling so maybe this isn't the right way to do it?

I'll put the comment on that pr